pkg/client/clientset_generated/release_1_2/typed/core/v1/fake/doc.go

```diff
@@ -14,5 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This file is generated by client-gen with arguments: --clientset-name=release_1_2 --input=[api/v1,extensions/v1beta1]
```

Comment:
We should move this auto-generated comment to doc.go, but that will be another PR.